metrics/linux/cpuusage.go

```diff
@@ -124,5 +124,12 @@ func (g *CPUUsageGenerator) collectProcStatValues() ([]float64, float64, uint, e
 		totalValues += values[i]
 	}
 
+	// Since cpustat[CPUTIME_USER] includes cpustat[CPUTIME_GUEST], subtract the duplicated values from total.
+	// https://github.com/torvalds/linux/blob/4ec9f7a18/kernel/sched/cputime.c#L151-L158
+	// https://github.com/mackerelio/mackerel-agent/issues/419
+	totalValues -= values[8]
```

Comment:
`values[8]` (cpu.guest) is available since Linux 2.6.24, according to `man 5 proc`.
Though our official support targets (CentOS >= 6, Debian >= 8) are all > Linux 2.6.24, but it's safe to check `values` has enough length.
(`metrics.linux.Generate()` checks length of collected stats.)